subsys/bluetooth/controller/ll_sw/lll_sync.h

```diff
@@ -66,3 +74,4 @@ void lll_sync_prepare(void *param);
 enum sync_status lll_sync_cte_is_allowed(uint8_t cte_type_mask, uint8_t filter_policy,
 					 uint8_t rx_cte_time, uint8_t rx_cte_type);
 extern uint16_t ull_sync_lll_handle_get(struct lll_sync *lll);
+extern struct lll_sync *ull_sync_lll_is_valid_get(struct lll_sync *lll);
```

Comment:
The name makes sense, but the `is_valid` part in the middle threw me off at first. 

Would
```suggestion
extern struct lll_sync *ull_sync_lll_get_valid(struct lll_sync *lll);
```

Make sense?

Reply:
`_is_valid` is widely used in Zephyr compare to `_get_valid`, as the return here is a reference hence I have used `_is_valid_get`. Keep the code as-is.